.github/workflows/main.yml

```diff
@@ -9,6 +9,38 @@ on:
   workflow_dispatch:
 
 jobs:
+  lint:
```

Comment:
The logs for the *Build package* step shows the name of the built artifacts is `pytest-asyncio-0.1.dev1+ga5205f2.tar.gz` and `pytest_asyncio-0.1.dev1+ga5205f2-py3-none-any.whl`. As far as I understand these are reused in the `deploy` stage later.

I expected them to be to have a version like `pytest_asyncio-0.16.1+gabcd1234…` not `0.1.dev1`.

The checkout action seems to do a shallow clone with `--no-tags`. Is it possible this prevents setuptools_scm from determining the correct version?

Reply:
`fetch-depth: 0` should fix it. Let's see